src/openforms/registrations/contrib/camunda/fields.py

```diff
@@ -0,0 +1,24 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import fields
+
+
+class NullField(fields.Field):
```

Comment:
Bit of a weird one since the value and type are the same, but we need to validate this.